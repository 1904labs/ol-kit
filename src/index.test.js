import * as ol_kit from './index' // eslint-disable-line camelcase

describe('ol-kit package', () => {
  it('should not export undefined', () => {
    Object.keys(ol_kit).forEach(exportKey => {
      expect(ol_kit[exportKey]).not.toBeUndefined()
    })

    expect(ol_kit).toMatchSnapshot()
  })
})
