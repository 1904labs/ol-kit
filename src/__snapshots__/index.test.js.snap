// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`ol-kit package should not export undefined 1`] = `
Object {
  "Compass": [Function],
  "Controls": [Function],
  "Map": [Function],
  "Popup": [Function],
  "PopupBase": [Function],
  "PopupDefaultInsert": [Function],
  "VectorLayer": [Function],
  "ZoomControls": [Function],
  "centerAndZoom": [Function],
  "connectToMap": [Function],
  "createMap": [Function],
  "createSelectInteraction": [Function],
  "updateMapFromUrl": [Function],
  "updateUrlFromMap": [Function],
}
`;
